<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="4N30FV2" failures="1" tests="1" name="Parallel Test" time="824.006" errors="0" timestamp="2024-11-20T15:51:04 IST">
  <testcase classname="testController.TestController" name="TestCaseController" time="822.542">
    <failure type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
at testController.TestController.TestCaseController(TestController.java:96)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:209)
at java.base/java.util.ImmutableCollections$Map1.<init>(ImmutableCollections.java:1105)
at java.base/java.util.Map.of(Map.java:1353)
at org.openqa.selenium.remote.DriverCommand.SWITCH_TO_WINDOW(DriverCommand.java:255)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.window(RemoteWebDriver.java:1119)
at keywords.Keywords.switchToParentWindow(Keywords.java:470)
... 23 more
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- TestCaseController -->
</testsuite> <!-- Parallel Test -->
